streams/readable-byte-streams/general.js

```diff
@@ -16,8 +16,17 @@ test(() => {
 test(() => {
   // Constructing ReadableStream with an empty underlying byte source object as parameter shouldn't throw.
   new ReadableStream({ type: 'bytes' });
+  // Constructor must perform ToString(type).
+  new ReadableStream({ type: { toString() {return 'bytes';} } });
+  new ReadableStream({ type: { toString: null, valueOf() {return 'bytes';} } });
 }, 'ReadableStream with byte source can be constructed with no errors');
 
+test(() => {
+  const rs = new ReadableStream({ type: 'bytes' });
+  assert_throws(new RangeError(), () => rs.getReader({ mode: { toString() {return 'byob';} } }));
+  rs.getReader({ mode: 'byob', notmode: 'ignored' });
+}, 'getReader({mode}) must only accept an actual string');
```

Comment:
This is a spec bug!! ToString should always be performed on string-expecting arguments. Thank you for finding.